src/__tests__/patients/allergies/Allergies.test.tsx

```diff
@@ -9,7 +9,7 @@ import thunk from 'redux-thunk'
 import { Router } from 'react-router'
 import { Provider } from 'react-redux'
 import Patient from 'model/Patient'
-import { Button, Modal, List, ListItem, Alert } from '@hospitalrun/components'
+import * as components from '@hospitalrun/components'
```

Comment:
I think here is not needed `* as components`, try with `import components from '@hospitalrun/components'`

Reply:
I tried this and got an error when trying to mock it.

Reply:
OK leave it as is and we will open a new issue to investigate